src/channels/pages/ChannelDetailsPage/ChannelDetailsPage.tsx

```diff
@@ -79,7 +79,7 @@ export const ChannelDetailsPage: React.FC<ChannelDetailsPageProps> = ({
     );
 
   return (
-    <Form onSubmit={onSubmit} initial={initialData}>
+    <Form confirmLeave onSubmit={onSubmit} initial={initialData}>
```

Comment:
Leaving only this form with confirmLeave for QA purposes, all the other places will be added in a separate pr